cmd/push_images.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/urfave/cli/v2"
```

Comment:
May need to run "go fmt" or one w/ imports (I forget which, my IDE does it) do separate stdlib imports from third parties.

Reply:
I'm using Goland set to do it, `go fmt` changes nothing here either. It looks like you have to tick an explicit setting in Goland for that, which I've done now